command/apply_destroy_test.go

```diff
@@ -198,20 +197,36 @@ func TestApply_destroyApproveYes(t *testing.T) {
 	defer os.RemoveAll(td)
 	defer testChdir(t, td)()
 
-	statePath := testTempFile(t)
+	// Create some existing state
+	originalState := states.BuildState(func(s *states.SyncState) {
```

Comment:
This previously attempted to destroy an empty state file, which doesn't require approval, so it always passed.